### ISSUE TITLE
Someone took my offer, not sure what went wrong?

### ISSUE THREAD

**Issue by 2smpl2** [bisq-network/bisq#2923]
java.lang.IllegalArgumentException

**Comment by dwknch** [comment]
I'm experiencing a similar issue, see #2924 and #2925, no solution so far, unfortunately

**Comment by a123b** [comment]
`java.lang.IllegalArgumentException` is not very informative without the full stack trace. Could you upload the bisq.log file?

Might be a duplicate of #2910 (and #2924).

**Comment by 2smpl2** [comment]
Jun-27 10:13:33.884 [JavaFX Application Thread] INFO  b.c.t.p.t.m.MakerSetupDepositTxListener: swapReservedForTradeEntry 
Jun-27 10:13:33.884 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: PublishTradeStatistics 
Jun-27 10:13:33.953 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: MakerVerifyTakerAccount 
Jun-27 10:13:33.954 [JavaFX Application Thread] INFO  b.c.t.TaskRunner: Run task: MakerVerifyTakerFeePayment 
Jun-27 10:13:33.954 [JavaFX Application Thread] INFO  b.c.t.p.TradeProtocol: Send AckMessage for DepositTxPublishedMessage to peer qnu4qfq6fon64o5a.onion:9999. tradeId=wdvibmd-146775a3-d93c-4cf3-b3ed-099970894b96-112, sourceUid=ef3a2f17-8e93-40e9-917c-2355263a22b1 
Jun-27 10:13:33.954 [JavaFX Application Thread] WARN  b.n.p.P2PService: We don't have the peer in our persisted peers so we don't know his capabilities. We decide to not sent the msg. peersNodeAddress=qnu4qfq6fon64o5a.onion:9999 
Jun-27 10:13:33.954 [JavaFX Application Thread] ERROR b.c.t.p.TradeProtocol: AckMessage for DepositTxPublishedMessage failed. Peer qnu4qfq6fon64o5a.onion:9999. tradeId=wdvibmd-146775a3-d93c-4cf3-b3ed-099970894b96-112, sourceUid=ef3a2f17-8e93-40e9-917c-2355263a22b1, errorMessage=We did not send the EncryptedMailboxMessage because the peer does not support the capability. 
Jun-27 10:13:33.997 [JavaFX Application Thread] INFO  b.c.t.p.t.m.MakerSetupDepositTxListener: We got the deposit tx already set from MakerProcessDepositTxPublishedMessage.  tradeId=wdvibmd-146775a3-d93c-4cf3-b3ed-099970894b96-112, state=MAKER_RECEIVED_DEPOSIT_TX_PUBLISHED_MSG 
Jun-27 10:13:33.997 [JavaFX Application Thread] INFO  b.c.t.p.t.m.MakerSetupDepositTxListener: swapReservedForTradeEntry
